java/ql/src/Security/CWE/CWE-020/OverlyLargeRange.ql

```diff
@@ -12,6 +12,7 @@
  *       external/cwe/cwe-020
  */
 
+private import semmle.code.java.AlertFiltering
```

Comment:
This is actually risky due to very obscure reasons - it can risk additional recomputation of cached predicates all through the library stack (import order _can_ matter for various tie-breakers in the optimiser). It's actually the query that doesn't completely follow best practice here as it doesn't start with `import java` (it only does that implicitly as the first-import chain actually eventually reaches `import java`, but your added import breaks that).
```suggestion
import java
```

Reply:
Applied.  Thanks!